sdk/vaa/payloads.go

```diff
@@ -74,8 +74,9 @@ var (
 	ActionDeleteWasmInstantiateAllowlist GovernanceAction = 5
 
 	// Gateway governance actions
-	ActionSetIbcComposabilityMwContract   GovernanceAction = 1
-	ActionSetTokenfactoryPfmDefaultParams GovernanceAction = 2
+	ActionScheduleUpgrade               GovernanceAction = 1
```

Comment:
Noting the change of governance VAA here. I expect that none of these have been produced for mainnet.